api/envoy/config/filter/http/compressor/v2/compressor.proto

```diff
@@ -45,4 +49,21 @@ message Compressor {
   // Runtime flag that controls whether the filter is enabled or not. If set to false, the
   // filter will operate as a pass-through filter. If not specified, defaults to enabled.
   api.v2.core.RuntimeFeatureFlag runtime_enabled = 5;
+
+  // A compressor library to use for compression. Currently only
+  // :ref:`envoy.filters.http.compressor.gzip<envoy_api_msg_config.filter.http.compressor.gzip.v2.Gzip>`
+  // is included in Envoy.
+  //
+  // This field is ignored when used in the context of the deprecated "envoy.filters.http.gzip"
+  // filter.
+  CompressorLibrary compressor_library = 6;
+}
+
+message CompressorLibrary {
+  // The name of the compressor library to instantiate for the filter.
+  string name = 1 [(validate.rules).string = {min_bytes: 1}];
```

Comment:
We can remove this for new extensions as the extension will be looked up via the typed_config.

Reply:
Yeap, removed.